jetson/power_logging/model/benchmark.py

```diff
@@ -154,10 +249,12 @@ def benchmark(args: argparse.Namespace) -> None:
 
         model_dir = f"{args.result_dir}/{args.model}"
         Path(model_dir).mkdir(exist_ok=True, parents=True)
-        file_name = f"{args.model}_tensorrt.json"
+        file_name = f"{args.model}_pytorch.json"
         file_path = f"{model_dir}/{file_name}"
         with open(file_path, "w", encoding="utf-8") as outfile:
             json.dump(results.model_dump(), outfile, indent=4)
+        with open(f"{model_dir}/{args.model}_layerwise_latency.json", "w") as layer_profiles_file:
```

Comment:
I am trying to understand how will these be saved as JSON? Is the following output correct?

```json
[
{
layer1: {'start_time': .., 'elapsed_time': ...} #run1
...
},
{
layer1: {'start_time': .., 'elapsed_time': ...} #run2
...
}
]
```

Reply:
Yep, that's correct